src/lib/middleware/validators.js

```diff
@@ -0,0 +1,39 @@
+export const categoryValidator = store => next => action => {
+  let actionType = action.type.split('_');
+  if (actionType[0] === 'CATEGORY'){
+    try{
+      console.log(action);
+
+      let result = next(action);
+
+      if (!action.payload.title) {throw new Error('no title provided')}

```

Comment:
This error message is not descriptive enough, it should explain what exactly did not get a title. Something like "Expenses needs a title provided" or something along those lines.